server/@types/forms/index.d.ts

```diff
@@ -3,6 +3,9 @@ declare module 'forms' {
     prisonNumber: string
     title?: string
     targetCompletionDate?: Date
+    'another-date-day'?: string
```

Comment:
Notwithstanding what I've commented on below re: the nunjucks view and manipulating arrays etc - could we change these field names please to `targetCompletionDate-xxx` ?
I think if you are only looking at this form class and see a field `targetCompletionDate` and `another-date-xxx` they are not obviously related. Someone might be forgiven for thinking they are completely different fields / have different purposes, which is not the case here

Reply:
Yep that makes sense - updated to be `targetCompletionDate-xxx`